src/apps/reservation-list/reservation-material/reservation-material.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useCallback, FC, MouseEvent } from "react";
+import fetchMaterial, {
+  MaterialProps
+} from "../../loan-list/materials/utils/material-fetch-hoc";
+import { ReservationType } from "../../../core/utils/types/reservation-type";
+import ReservationInfo from "./reservation-info";
+import fetchDigitalMaterial from "../../loan-list/materials/utils/digital-material-fetch-hoc";
+import MaterialInfo from "../../loan-list/materials/stackable-material/material-info";
+
+export interface ReservationMaterialProps {
+  reservation: ReservationType;
+}
+
+const ReservationMaterial: FC<ReservationMaterialProps & MaterialProps> = ({
+  material,
+  reservation
+}) => {
+  function stopPropagationFunction(e: Event | MouseEvent) {
+    e.stopPropagation();
+  }
+
+  useEffect(() => {
+    document
+      .querySelector(".list-reservation a")
+      ?.addEventListener("click", stopPropagationFunction, true);
+
+    return () => {
+      document
+        .querySelector(".list-reservation a")
+        ?.removeEventListener("click", stopPropagationFunction, true);
+    };
+  }, []);
```

Comment:
Could you please elaborate what is going here?

A component reaching outside its scope through `document.querySelector()` does not seem like a good idea. As I see it we will end up with many event listeners on each element. I wonder what has lead you here.

If there are links within the child component that we want to override could be pass an onClick function to the child components instead and let them handle things?